source/read-write-concern/read-write-concern.rst

```diff
@@ -380,6 +381,38 @@ Drivers SHOULD NOT parse server replies for "writeConcernError" in generic comma
 (Reporting of writeConcernErrors is more complex for bulk operations,
 see the Bulk API Spec.)
 
+writeConcernError Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The set of possible writeConcernErrors is quite large because they can include
+errors caused by shutdown, stepdown, interruption, maxTimeMS, and wtimeout.
+This section attempts to list all known error codes that may appear
+within a writeConcernError but may not be exhaustive. Note that some errors
+have been abbreviated:
+
+- ``{ok:1, writeConcernError: {code: 91, codeName: "ShutdownInProgress"}}``
+- ``{ok:1, writeConcernError: {code: 189, codeName: "PrimarySteppedDown"}}``
+- ``{ok:1, writeConcernError: {code: 11600, codeName: "InterruptedAtShutdown"}}``
+- ``{ok:1, writeConcernError: {code: 11601, codeName: "Interrupted"}}``
+- ``{ok:1, writeConcernError: {code: 11602, codeName: "InterruptedDueToReplStateChange"}}``
+- ``{ok:1, writeConcernError: {code: 64, codeName: "WriteConcernFailed", errmsg: "waiting for replication timed out", errInfo: {wtimeout: True}}}``
+- ``{ok:1, writeConcernError: {code: 64, codeName: "WriteConcernFailed", errmsg: "multiple errors reported : {...} at shardName1 :: and :: {...} at shardName2"}}`` [#]_
+- ``{ok:1, writeConcernError: {code: 50, codeName: "MaxTimeMSExpired"}}``
+- ``{ok:1, writeConcernError: {code: 100, codeName: "UnsatisfiableWriteConcern", errmsg: "Not enough data-bearing nodes"}}``
+- ``{ok:1, writeConcernError: {code: 79, codeName: "UnknownReplWriteConcern"}}``
+
+Note also that it is possible for a writeConcernError to be attached to a
+command failure. For example:
```

Comment:
Is a `writeConcernError` being attached to a command failure only relevant for mongos? You only present a single example here, which includes a footnote to the longer explanation. If so, I think we should just combine the footnote and this sentence into a single paragraph about mongos behavior.

Reply:
Oh, I didn't realize that there were multiple footnotes. Disregard.